### ISSUE TITLE
Slight tweak to `focusHighlightForeground` variable colour to allow for more contrast/accessibility

### ISSUE THREAD

**Issue by gpeuc** [microsoft/vscode#152184]
Hello everyone,

we at SAP are building some cool things inside VS Code. One of our main drivers when designing things is that they should look like they belong inside VS Code. To achieve that we are borrowing heavily 🥷 from the default VS Code components. Basically, we are scavenging anything that we see natively built in VS Code, and we replicate it.

One of such elements is a complex dropdown, which features search, typeahead, highlights of the typed stuff, etc. It is made to replicate VS Code native Command Palette.

Of course, we are using semantically correct variables to paint everything, and to adhere to theming.

We have noticed one slight issue with colour values in the variables used to paint the Command Palette.

## The issue 

The main issue can be described simply by noticing that blue-on-blue highlight isn't contrasty enough under most popular themes (Dark+ and Light+).

Essentially, for Dark+, we have `18A3FF` colour of the focused blue text being placed on top of `094771` dark blue background. This gives a contrast ratio of `3.58:1` which fails accessibility contrast check.

I have proposed below to update the focus colour to `52BAFF`, which still keeps it blue, but offers **4.56:1** contrast ratio.

Similar things happens in Light+, and I have proposed the colour bump to alleviate that issue.

## The samples

In the following picture you can see the current rendering of Command Palette, followed by our replica of that same thing as a dropdown/combobox, and then some colour blobs to closer see the exact situation.

<img width="814" alt="focus colours" src="https://user-images.githubusercontent.com/44162846/173820981-04271c25-cddb-4ea1-a41e-b539af3419df.png">

## The debate

The main question here is - would you be in a position to update the variables so that the blue-on-blue scenario for Dark+ and Light+ themes is more accessible? 

Now, you could easily say "well, this is why we have High Contrast themes", and you would be totally correct, but I believe we don't have to immediately pull out the big guns. We do have people, like myself, who are in a ... khm ... certain age that begins with 40 and ends with ever increasing number ... who still appreciate the beauty of regular themes, but a slight bump in contrast would mean a lot, without forcing me to immediately go High Contrast theme.

Thank you,
All the love from Dublin, Ireland and SAP

``
- VS Code Version: Version: 1.68.0 (Universal) (latest as of this day)
- OS Version: MacOS 12.4 (latest as of this day)

**Comment by misolori** [comment]
Thanks for the feedback and possible solutions ❤️  cc @joaomoreno @TylerLeonhardt @isidorn we might be able to roll this one into https://github.com/microsoft/vscode/issues/151577 since it's also tackling color contrast for the list widget, which the quick pick uses.

**Comment by gpeuc** [comment]
@misolori This would be absolutely brilliant. Thanks for the consideration. Of course, the hex values I've provided are simply the first hex values in that same hue, only made brighter, that take the contrast over the 4.5 target. If you already have a better set of hex values which solve the blue-on-blue problem, it's your call. You are the chief here, I'm merely a consumer (a consumer with suggestions, but hey, still just a consumer).

**Comment by misolori** [comment]
I've got a PR up at https://github.com/microsoft/vscode/pull/152351 that will address this, here are the updated colors:

![image](https://user-images.githubusercontent.com/35271042/174114026-d5f586d3-7155-4a19-bdfb-c241e42b7c52.png)

![image](https://user-images.githubusercontent.com/35271042/174114007-b51beff7-cede-43f6-9b88-da1e77fe1c43.png)
